build.gradle

```diff
@@ -202,8 +202,3 @@ buildscript {
         classpath 'org.eclipse.jgit:org.eclipse.jgit:2.0.0.201206130900-r'
     }
 }
-
-task wrapper(type: Wrapper) {
```

Comment:
Newer version of Gradle have the `wrapper` task *already included*. Upgrading to a new Gradle wrapper version is as easy as invoking this command:

    ./gradlew wrapper --gradle-version 3.5 --distribution-type all

Note: The distribution type defaults to `bin`, and for Android projects the default and recommended distribution is `all`, as it adds additional support (sources) to the IDE while working on Gradle scripts. The version/distribution only needs to be downloaded *once* for all projects and is cached system-wide (usually in `~/.gradle`).